### PR TITLE
Style overhaul

### PR DESCRIPTION
**Pull request by tkenakka**
- Meeting background colours and meeting status colours still in prototyping stage.
- Deployed to test server

### CODE REVIEW

File: app/directives/meetings/admeetings.directive.js [hunk 1]
```diff
@@ -103,17 +103,15 @@ angular.module('dashboard')
...
                 if ("objects" in self.responseData) {
                     $log.debug("adMeetings: getMeetings done: " + self.responseData.objects.length);
                 }
             }, function(error) {
                 $log.error("adMeetings: getMeetings error: " + JSON.stringify(error));
-                self.loading = false;
                 self.mtgErr = error;
             }, function(/*notify*/) {
-                self.loading = true;
+                self.loading = false;
```
Comment (evidentti) [inline app/directives/meetings/admeetings.directive.js]: Is this at correct place? Should it be in finally?

File: app/styles/main.scss [hunk 2]
```diff
@@ -408,34 +467,16 @@ body {
...
-
-    .db-colheader-btn-dummy {
-        @extend .db-colheader-btn;
-        opacity: 1.0 !important;
-        outline: none !important;
+    @media screen and (max-width: 767px) {
+        .db-btn-list-topleft-xs { border-top-left-radius: $btn-radius !important; }
+        .db-btn-list-topright-xs { border-top-right-radius: $btn-radius !important; }
+        .db-btn-list-topleft-sm {}
+        .db-btn-list-topright-sm {}
```
Comment (evidentti) [inline app/styles/main.scss]: Is empty declaration needed? It will give a warning.

File: app/styles/main.small.scss [hunk 3]
```diff
@@ -75,4 +71,9 @@ $btn-radius: 8px;
             }
         }
     }
+
+    .db-btn-list-topleft-xs {}
```
Comment (evidentti) [inline app/styles/main.small.scss]: Empty declaration.

**Comment by evidentti** [comment]
Proposal headers are not shown in mobile. (Julkaisemattomat, Julkaistut)
LGTM after comments.

**Comment by tkenakka** [comment]
- Review comments fixed and merged
- Rebased no latest devel & fixed conflicts due to latest merges

**Pull request merged**